app/services/clients/cardid.client.js

```diff
@@ -1,16 +1,27 @@
 'use strict'
 
 // Local dependencies
-const baseClient = require('./base.client/base.client')
+const { Client } = require('@govuk-pay/pay-js-commons/lib/utils/axios-base-client/axios-base-client')
+const { configureClient } = require('./base/config')
 
 // Constants
 const CARD_URL = process.env.CARDID_HOST + '/v1/api/card'
+const client = new Client('cardid')
+configureClient(client, CARD_URL)
 
 /**
  *
  * @param args
  *
  * @returns {Request}
  */
-exports.post = (args) => baseClient.post(CARD_URL, args, null)
+
+exports.post = async (args) => {
+  try {
```

Comment:
We don't need a try / catch here.

The calling method wraps this in a promise anyway and will deal with the error.

Reply:
As discussed, this is no longer required.
As Axios behaves a bit differently to request library.